test_autoflake.py

```diff
@@ -1224,6 +1224,43 @@ def test_exclude(self):
         finally:
             shutil.rmtree(temp_directory)
 
+    def test_with_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        try:
+            with open(
+                 os.path.join(temp_directory, '__init__.py'), 'w') as output:
```

Comment:
visually , this indentation is _wrong_.  pycodestyle may not show it as an error , but that is a deficiency in pycodestyle IMO, and probably an actual bug, maybe even a regression.  pep8 is quite clear about this, at least in the spirit of the document, if not in the letter of it.

the simplest way to make this much better is to move `os.path.join(temp_directory, '__init__.py')` out of this line, and put it into a variable before this line.

Also notice that below you use `temp_directory + '/__init__.py'` , so that should likely also be using the same variable.  I assume on Windows autoflake accepts Windows directory slashes correctly. (If not, that is an issue which does need to be raised.)

Reply:
Fixed! I don't have a Windows computer on me currently, so I unfortunately cannot confirm if autoflake accepts Windows directory slashes correctly.